server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/TasksRoutesTest.java

```diff
@@ -140,12 +140,16 @@ public void listShouldListTaskWhenStatusFilter() {
     }
 
     @Test
-    public void listShouldReturnEmptyWhenNonMatchingStatusFilter() {
+    public void listShouldReturnEmptyWhenNonMatchingStatusFilter() throws Exception {
+        CountDownLatch inProgressLatch = new CountDownLatch(1);
         taskManager.submit(() -> {
```

Comment:
why is this TaskManager unable to provide a Future ?

Reply:
A future won't help here... As  in progress task is required.

Future would only help to ensure "something is done". Not that "something is not".

However I agree these latch are complicated. We of course should keep some "non finished tasks in the tests" but we might want to "*do some simpler tests where we can afford it.*"

Reply:
ok